Rubberduck.Parsing/ComReflection/ComBase.cs

```diff
@@ -13,6 +13,8 @@ public interface IComBase
         ComDocumentation Documentation { get; }
         string Name { get; }
         DeclarationType Type { get; }
+        IComBase  Parent { get; }
+        ComProject Project { get; }
```

Comment:
I wonder if we'd benefit having a `IEnumerable<IComBase> Children` to support two-way traversal in a general manner?

Reply:
That might not be a bad idea @future, I didn't give it much thought because the current use case is always down the tree.